components/automate-ui/src/app/entities/notification_rules/notification_rule.action.ts

```diff
@@ -26,7 +35,64 @@ export class GetNotificationRulesFailure implements Action {
   constructor(public payload: HttpErrorResponse) { }
 }
 
+export class GetNotification implements Action {
+  readonly type = NotificationRuleActionTypes.GET;
+  constructor(public payload: { id: string }) { }
+}
+
+export class GetNotificationSuccess implements Action {
+  readonly type = NotificationRuleActionTypes.GET_SUCCESS;
+  constructor(public payload: NotificationRule) { }
+}
+
+export class GetNotificationFailure implements Action {
+  readonly type = NotificationRuleActionTypes.GET_FAILURE;
+  constructor(public payload: HttpErrorResponse, public id: string) { }
```

Comment:
asking for my own learning because I haven't seen this before.  Is it necessary to mark `id` as public?
in other words, is
```
constructor(public payload: HttpErrorResponse, public id: string) { }
```

different than this:

```
constructor(public payload: HttpErrorResponse, public id: string) { }
```